### ISSUE TITLE
Filtering Support

### ISSUE THREAD

**Issue by krische** [djangonauts/django-rest-framework-hstore#9]
It would be nice to have some basic filtering support of the hstore fields with Django Rest Framework.

**Comment by nemesifier** [comment]
@krische, contributions are welcome, if you manage to implement something interesting please send a pull request with the code so we can discuss further.

**Comment by krische** [comment]
Yeah I was starting to play around with creating an `HStoreBackend` filtering class. I was thinking thinking users could specify their filter query parameter as `FIELD.KEY=VALUE`. That should at least give some kind of basic filtering for hstore fields that are just storing key/value pairs.

**Comment by daviwesley** [comment]
@krische Did u implement the filter? I also need some filter capabilities in my API to HStore
